microcosm_flask/conventions/crud_adapter.py

```diff
@@ -46,3 +46,22 @@ def update(self, **kwargs):
         identifier = kwargs.pop(self.identifier_key)
         model = self.store.model_class(id=identifier, **kwargs)
         return self.store.update(identifier, model)
+
+    def update_batch(self, **kwargs):
```

Comment:
The CRUD adapter's update batch function is probably too simplistic to be useful; we'll have to learn from experience.